src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -549,11 +549,11 @@ try
 
     if (multiClickMapper == 3)
     {
-        _terminal->MultiClickSelection(cursorPosition / fontSize, ::Terminal::SelectionExpansionMode::Line);
+        _terminal->MultiClickSelection(cursorPosition / fontSize, InternalSelectionExpansion::Line);

```

Comment:
I'm leave a note behind here saying that once `PublicTerminalCore` is moved to directly consume winrt, we can get rid of `InternalSelectionExpansion` entirely, and just use `winrt::Microsoft::Terminal::Core::SelectionExpansion`

Reply:
#6106 or #6999